tests/integration/defs/accuracy/test_llm_api_autodeploy.py

```
@@ -1019,13 +1019,27 @@ class TestQwen3_5_397B_MoE(LlmapiAccuracyTestHarness):
     MODEL_NAME_SMALL = "Qwen/Qwen3.5-35B-A3B"
     MODEL_PATH_SMALL = hf_id_to_local_model_dir(MODEL_NAME_SMALL)
+    WORLD_SIZE = 8
+    # Config files driving this test. Resolved directly from the configs
+    # directory rather than the model registry so the test stays valid
+    # regardless of which models are enabled in models.yaml.
+    YAML_EXTRA = [
+        'dashboard_default.yaml',
+        'world_size_8.yaml',
+        'qwen3.5_moe_400b.yaml',
+        'enable_sharder_ir.yaml',
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)